pyproject.toml

```
@@ -22,6 +22,7 @@ classifiers = [
 include = ["strawberry/py.typed"]
 
 [tool.poetry.urls]
+"Changelog" = "https://github.com/strawberry-graphql/strawberry/blob/main/CHANGELOG.md"
```

```suggestion
"Changelog" = "https://strawberry.rocks/changelog"
```

I made a redirect, so in case we put the changelog on our site we won't have to update the pyproject.toml 😊